accepted/0000-caasp-integration.md

```diff
@@ -0,0 +1,366 @@
+    - Assign the CaaS Platform channels assigned via Salt (SUSE CaaS Platform Extension to SUSE Linux Enterprise 15 and SUSE Containers Module 15 SP1 x86_64), whereas the AutoYaST case will be already covered during provisioning.
+    - De-register the machine from SUSE Manager (if it is registered)
+
+  - operate on the Management Node:
+    - Salt will invoke `skuba` (using `cmd.run`) to bootstrap the first control plane or make the selected node join the cluster. The user must specify which user to use for passwordless login (in this case `sles`) and whether to use `sudo` or not.
+
+    Example:
+
+    ```
+    bootstrap_first_control_plane:
+      cmd.run:
+        - name: skuba node bootstrap --user sles --sudo --target <node IP> <node name>
+    
+    bootstrap_additional_control_plane:
+      cmd.run:
+        - name: skuba node join --role master --user sles --sudo --target <node IP> <node name>
+    
+    bootstrap_worker:
+      cmd.run:
+        - name: skuba node join --role worker --user sles --sudo --target <node IP> <node name>
+    ```
+
+    SUSE Manager will check the return code of the `skuba` command and, in case of errors, show the raw output.
+
+## Manage a cluster using SUSE Manager
+
+SUSE Manager offers to show the status of the running cluster by either accessing the cluster directly or via the Management Node.
+In this regard, the cluster provider should provide a way to expose the cluster status (e.g. running nodes, health of each node, monitoring metrics...) to SUSE Manager.
+
+### Add and removal of nodes to the cluster
+
+SUSE Manager also offers the possibility of adding and removing nodes to the cluster:
+
+- To add a node from scratch, an AutoYaST profile can be reused
+- To remove a node, SUSE Manager must remove the node from the cluster using the Management Node and issuing the proper procedure to restrict a node off the cluster
+
+SUSE Manager should be instructed to perform additional actions depending on the role of added/removed node (e.g. control plane, worker, load balancer, ...).
+
+### Cluster upgrade
+
+If every node of the cluster is not directly registered to SUSE Manager, SUSE Manager should offer the possibility to use the Management Node to trigger an update of every node in the cluster, following the standard procedure of upgrading predisposed by the cluster provider.
+
+Also, SUSE Manager must coordinate the Management Node to issue a correct upgrade procedure of the product (e.g. from CaaS Platform version 3 to 4).
+
+### Application deployment
+
+SUSE Manager can also offer the possibility of deploying an application on top of the cluster deployed (e.g. Kubernetes deployment descriptor to deploy a Kubernetes operator). In this case, the cluster provider must provide the Management Node of the necessary tooling to deploy an application using an API. SUSE Manager makes use of the Management Node to deploy a provided application.
```

Comment:
Isn't all of that about having access to the Kubernetes API? Practically speaking you would need a `kubectl` configuration file.

Reply:
This part is the "Integrating with a cluster provider" generic section, in response to https://github.com/uyuni-project/uyuni-rfc/pull/10#issuecomment-520243504
I will add `kubectl` point to the "Run other products in the cluster" section